tesseract-core/src/dataframe.rs

```diff
@@ -65,6 +65,119 @@ impl Column {
     pub fn column_data(&mut self) ->&mut ColumnData {
         &mut self.column_data
     }
+
+    /// DataFrame columns can come in many different types. This function converts
+    /// all data to a common type (String).
+    pub fn stringify_column_data(&self) -> Vec<String> {
+        return match &self.column_data {
+            ColumnData::Int8(v) => {
+                let mut t: Vec<i8> = v.iter().map(|&e| e.clone() as i8).collect();
+                t.sort();
```

Comment:
leaving a note here for tracking, after discussing with @MarcioPorto in person; sort on dataframe cols will be pulled into a separate method.